js/inspector.ts

```diff
@@ -0,0 +1,153 @@
+import type { RenderProps } from '@anywidget/types';
+import { css, html, nothing, TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import { legacyStyles } from './ipywidgets_styles';
+import { LitWidget } from "./lit_widget";
+import type { Node } from './tree_node';
+
+import './container';
+import './tree_node';
+
+export interface InspectorModel {
+    hide_close_button: boolean;
+    expand_points: boolean;
+    expand_pixels: boolean;
+    expand_objects: boolean;
+    point_info: { [key: string]: any };
+    pixel_info: { [key: string]: any };
+    object_info: { [key: string]: any };
+}
+
+export class Inspector extends LitWidget<InspectorModel, Inspector> {
+    static get componentName(): string {
+        return `inspector-widget`;
+    }
+
+    static styles = [
+        legacyStyles,
+        css`
+            .checkbox-container {
+                align-items: center;
+                display: flex;
+                gap: 8px;
+                height: 32px;
+            }
+
+            .spacer {
+                width: 8px;
+            }
+
+            .object-browser {
+                max-height: 300px;
+                min-width: 290px;
+                overflow: scroll;
```

Comment:
overflow: auto, otherwise the scrollbar will always be present.

![image](https://github.com/user-attachments/assets/b221c999-69be-48aa-82b4-b467bf678b61)